docs/core/deploying/index.md

```diff
@@ -1,98 +1,173 @@
 
-FDD and SCD deployments use separate host executables, so you can sign a host executable for an SCD with your publisher signature.
+Once you acquire the tools, you're ready to build your app. Pick the approach that corresponds to whether in the previous step you chose to use an IDE like Visual Studio or Visual Studio for Mac or the CLI.
 
-### Why deploy a self-contained deployment?
+[Build your first .NET Core app using the Command Line Interface (CLI)](../tutorials/using-with-xplat-cli.md)
 
-Deploying a Self-contained deployment has two major advantages:
+[Build your first .NET Core app using Visual Studio](../tutorials/with-visual-studio)
 
-- You have sole control of the version of .NET Core that is deployed with your app. .NET Core can be serviced only by you.
+[Build your first .NET Core app using Visual Studio Code](https://code.visualstudio.com/docs/languages/dotnet)
 
-- You can be assured that the target system can run your .NET Core app, since you're providing the version of .NET Core that it will run on.
+Any of these options will allow you to create various types of .NET Core applications including ASP.NET Core, WPF, and Windows Forms applications.
 
-It also has a number of disadvantages:
 
-- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+<br/>
 
-- The size of your deployment package is relatively large, since you have to include .NET Core as well as your app and its third-party dependencies.
+### Step 2: Publish your app
 
-  Starting with .NET Core 2.0, you can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/globalization-invariant-mode.md). Ordinarily, .NET Core on Linux relies on the [ICU libraries](http://icu-project.org) for globalization support. In invariant mode, the libraries are not included with your deployment, and all cultures behave like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
+You can publish your app in one of two configurations - as a [Framework Dependent Deployment (FDD)](publish.md#Framework-Dependent-Deployment-FDD) or as a [Self-Contained Deployment (SCD)](publish.md#Self-contained-deployment-SCD).
 
-- Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
+For both of the above configurations, you can publish the app either as a set of loose assemblies for either `xcopy` or package up for deployment. Alternatively, you can publish your app as a single file executable.
```

Comment:
I don't get this paragraph.  What do you mean by "you can publish the app either as a set of loose assemblies for either `xcopy` or package up for deployment"  This is more about packaging isn't it?  You talk about this in the next section so why is it discussed here?  Also both of these also apply to single file executables as well don't they?

Reply:
What I'm trying to say is at a high level there are 2 options (FDD and SCD), and within each there are 2 sub-options - publish as loose assemblies or as a single file executable. I've rephrased, this, let me know if this version makes more sense.